MenuRenderer, OverflowMenu: Ensure max height (#1679)
seek-oss/braid-design-system: .changeset/friendly-trains-explain.md

---
'braid-design-system': patch
---

---
updated:
  - MenuRenderer
  - OverflowMenu
---

> **MenuRenderer, OverflowMenu**: Fix bug, ensure the `Menu` size is limited by a maximum height.

Reviewer: ```suggestion
**MenuRenderer, OverflowMenu**: Ensure the menu has a maximum height.
```
